parsers/prefix-by/README.md

```diff
@@ -0,0 +1,99 @@
+# Prefix By
+
+## Description
+This parser helps you concatenate two strings.
+
+Learn more about how to configure Specify in the API documentation: [https://specifyapp.com/developers](https://specifyapp.com/developers).
+
+## Interface
+
+```ts
+interface parser {
+  name: 'prefix-by';
+  options?: {
+    key?: string;
+    prefix: string;
+    types?: Array<string>;
```

Comment:
Wouldn't it be better to name the key `applyOn` or `applyOnTokens`? I really like the idea

Reply:
So I prefer `applyOn `

Reply:
@ChucKN0risK please rename the option

Reply:
I don't mind doing this but we should also update the `suffix-by` parser as it also has the same `types` property. This will cause a breaking change though.

WDYT?

Reply:
Parsers are isolated, and options are scoped. You can safely use `applyOn` here.

If you want to add the option applyOn on suffix-by. Feel free. It's another parser so another subject.

Reply:
Done for `prefix-by` ✅